R/format_geoparquet.R

```diff
@@ -0,0 +1,16 @@
+format_geoparquet <- tar_format(
+  read = function(path) {
+    arrow::read_parquet(path) |>
+      dplyr::mutate(dplyr::across(wk::is_handleable, wk::as_wkb))
```

Comment:
In order to roundtrip a SpatVector with geometry from parquet, need `vect(geom="geometry")`
```suggestion
      dplyr::mutate(dplyr::across(wk::is_handleable, wk::as_wkb)) |>
      terra::vect(geom = "geometry")
```